custom_model_runner/datarobot_drum/drum/common.py

```diff
@@ -16,6 +16,7 @@
 CLASS_LABELS_ARG_KEYWORD = "class_labels"
 TARGET_TYPE_ARG_KEYWORD = "target_type"
 X_TRANSFORM_KEY = "X.transformed"
+RESPONSE_PREDICTIONS_KEY = "predictions"
```

Comment:
We have REGRESSION_PRED_COLUMN. Do we need to have another one, not capitalized?

Reply:
it seems the key from the prediction server comes back keyed to a lower-case 'predictions'...so we presumably need both keys (although this could probably be renamed to be clearly about pred server output?) unless we want to standardize to either capital or lowercase in both situations

Reply:
got it, it's RESPONSE, lets keep it